internal/stacksapi/fail.go

```diff
@@ -0,0 +1,30 @@
+package stacksapi
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+type FailJobRequest struct {
+	StackKey    string // The key to call the stack. Required.
+	JobUUID     string // The uuid of a job
+	ExitStatus  int    `json:"exit_status,omitempty"`
```

Comment:
```suggestion
	ExitStatus  int    `json:"exit_status,omitzero"`
```

is probably a little clearer in intent (though not technically different in this case), and will make it easier for us to migrate to JSONv2 when (when!!!!) we do that